### PR TITLE
Add project id to project synchronization dropdown

### PR DESCRIPTION
**Pull request by ksierks**
## Description of changes
Added project id to project dropdown.

image

## Related issue
An extension of #1352

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* ~[ ] Tests added (or description of how to test) for any new features.~
* ~[ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/webapp/resources/js/types/irida/index.d.ts [hunk 1]
```diff
@@ -131,6 +131,7 @@ declare namespace IRIDA {
     analysisPriority: string;
     users: User[];
     analysisTemplates: string[]; // TODO (Josh - 6/7/22): What should this be
+    url: string;
```
Comment (joshsadam) [inline src/main/webapp/resources/js/types/irida/index.d.ts:134]: Do we want to name this `remoteUrl` to directly map to the server Project model?  That is what it is used for right?

**Review by joshsadam — CHANGES_REQUESTED** [review]
Just one comment, and it is a question.  If it is the remoteUrl I think we should name it that.

File: src/main/webapp/resources/js/types/irida/index.d.ts (hunk 1 above)
Comment (ksierks) [reply src/main/webapp/resources/js/types/irida/index.d.ts]: Ok, I was just doing what was easiest for me in the moment. Please see b9fae817b6f651d44e6e9ada08fd052af4814eef.

**Review by joshsadam — APPROVED** [review]
This looks good, thanks

**Pull request merged**